src/main/resources/templates/detect-ps.ps1

```diff
@@ -232,11 +232,30 @@ function Get-DetectJar ($DetectFolder, $DetectSource, $DetectVersionKey, $Detect
 
     if ($DetectSource -eq "") {
         if ($DetectVersion -eq "") {
-            $DetectVersionUrl = "https://repo.blackduck.com/api/storage/bds-integrations-release/com/blackduck/integration/detect?properties=" + $DetectVersionKey
+            $detectVersionKeySplit = $DetectVersionKey.split("_") # Split the key provided, as an example key would be of form: DETECT_LATEST_9
+            $detectVersionChar = $detectVersionKeySplit[-1]
+            $detectVersionNumber = [int]$detectVersionChar
+
+            # If major version is 9 or less, than download from synopsys location or else blackduck
```

Comment:
I see we are downloading from repo.blackduck in both cases. Seems like that or the comment should be changed.

Reply:
Sorry the comment is confusing, from synopsys I meant com/synopsys/integration location. For detect10 as discussed in standup we would be downloading all artifacts from repo.blackduck as we have put out the announcements for whitelisting the urls.